src/.babelrc

```diff
@@ -0,0 +1,4 @@
+
+{
+    "presets": ["env", "react"]
+  }
```

Comment:
@bellangowi i do not think this file is necessary. unless you have a reason to have it here?

Reply:
since there is enzyme, babel and all its ok.